assets/src/edit-story/types.js

```diff
@@ -70,13 +69,15 @@ StoryPropTypes.layer = PropTypes.shape(StoryLayerPropsTypes);
 
 StoryPropTypes.elements = {};
 
-StoryPropTypes.elements.image = PropTypes.shape({
+StoryPropTypes.elements.shape = PropTypes.shape({
   ...StoryElementPropsTypes,
-  src: PropTypes.string.isRequired,
+  backgroundColor: PropTypes.string,
+  backgroundImage: PropTypes.string.isRequired,
   origRatio: PropTypes.number.isRequired,
   scale: PropTypes.number.isRequired,
   focalX: PropTypes.number,
   focalY: PropTypes.number,
+  mask: StoryPropTypes.mask.isRequired,
```

Comment:
IMHO, we should allow `mask` for absolutely any type of element. If it's not exposed in some cases - that's fine. But it's inherent to any element type.